### PR TITLE
Upgrade prettycli to 1.4.3 (chalk 2.0.1)

### PR DESCRIPTION
**Pull request by siddharthkp**
Solves https://github.com/siddharthkp/bundlesize/issues/89

Update from https://github.com/siddharthkp/prettycli/pull/5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "github-build": "^1.2.0",
     "glob": "^7.1.2",
     "gzip-size": "^3.0.0",
-    "prettycli": "^1.3.0",
+    "prettycli": "1.4.3",
```
Comment (vs1682) [inline package.json]: @siddharthkp Don't wanna put a caret or tilde sign.

**Comment by palashmon** [comment]
Fixed a small typo in description.

File: package.json (hunk 1 above)
Comment (siddharthkp) [reply package.json]: My bad! Fixed.

Comment (vs1682) [reply package.json]: Cool.

**Comment by palashmon** [comment]
@siddharthkp I think we should update the repo version as well, as 4 new PRs have been merged after last release of version 0.14.1

**Comment by siddharthkp** [comment]
Will do!

**Comment by palashmon** [comment]
Awesome :sunglasses:

**Pull request merged**